client/js/lounge.js

```diff
@@ -275,7 +286,7 @@ $(function() {
 
 			chat.find(".chan.active .chat").trigger("msg.sticky"); // fix growing
 		})
-		.tab(completeNicks, {hint: false})
+		.tab(completeNicks(false), {hint: false})
```

Comment:
It's unnecessary complexity and the way it's used does not warrant a "dynamic" function.